### PR TITLE
fix: nomidotwatcher pod should not restart

### PR DESCRIPTION
**Pull request by yjkimjunior**
fixes: https://github.com/paritytech/Nomidot/issues/116

weell it doens't really fix it, but it at least makes the problem a lot more easy to discover and recover from.

### CODE REVIEW

**Comment by yjkimjunior** [comment]
actually this should be a Job not a Deployment

File: back/node_watcher/kubernetes/staging/nomidotwatcher-job.yaml [hunk 1]
```diff
@@ -8,23 +8,20 @@ metadata:
...
+      restartPolicy: Never
       containers:
         - name: nomidotwatcher
-          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:73f32773eb4e397e9f4cc7028945750f8f1116c7e751179120300a18413797ee
+          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:23bda9bd84369d74bc4f9a299649f388e4676bf37700c82a2df2cb577808d00a
           env:
             - name: PRISMA_ENDPOINT
               value: http://10.0.9.43:4466
+            - name: START_FROM
+              value: '38000'
```
Comment (yjkimjunior) [inline back/node_watcher/kubernetes/staging/nomidotwatcher-job.yaml]: this way I don't have to create and push a whole new image everytime i want to recover from a failure from a particular branch. just need to do `kubectl replace -f ...`

File: back/node_watcher/kubernetes/staging/nomidotwatcher-job.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: batch/v1
+kind: Job
```
Comment (yjkimjunior) [inline back/node_watcher/kubernetes/staging/nomidotwatcher-job.yaml]: this way it runs the script to completion, or tries to. if something fails then it won't strrt over and waste cpu

File: back/node_watcher/kubernetes/nomidotwatcher-job.yaml [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
       containers:
         - name: nomidotwatcher
-          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:73f32773eb4e397e9f4cc7028945750f8f1116c7e751179120300a18413797ee
+          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:22592d41b85880683b595de06f3b399eef3b42750baf02f6afc73e85130152ea
+          imagePullPolicy: Always
           env:
             - name: PRISMA_ENDPOINT
               value: http://10.0.9.43:4466
+            - name: START_FROM
+              value: '37921'
```
Comment (amaurymartiny) [inline back/node_watcher/kubernetes/nomidotwatcher-job.yaml]: Are you trying to spawn multiple nomidot watchers? Why start at this block?

Comment (yjkimjunior) [reply back/node_watcher/kubernetes/nomidotwatcher-job.yaml]: there was a bug with proposals around there, so process would exit, pod restarts from where the last job started (i had hardcoded for the previous version). It's a workaround just to get a full sync at least once so we have one to work with while figuring out a more sustainable way https://github.com/paritytech/Nomidot/issues/109

**Comment by yjkimjunior** [comment]
closing to make a clean PR. too many things slipped in here already

**Pull request closed without merging**